index.js

```diff
@@ -93,23 +99,23 @@ class Plugin extends MiniAccountsPlugin {
   _validatePaychanDetails (paychan) {
     const settleDelay = paychan.settleDelay
     if (settleDelay < util.MIN_SETTLE_DELAY) {
-      debug(`incoming payment channel has a too low settle delay of ${settleDelay.toString()}` +
+      this._log.debug(`incoming payment channel has a too low settle delay of ${settleDelay.toString()}` +
```

Comment:
I think all of these validation checks should be warns because they suggest someone is trying to game us